src/content/docs/alerts-applied-intelligence/notifications/intro-notifications.mdx

```diff
@@ -10,16 +10,28 @@ metaDescription: "Notifications are where you integrate your New Relic data with
 ---
 
 Notifications are a consolidation of the different ways to send notification-events to third-party services, such as Slack, Jira, ServiceNow, and email. You can also use webhooks to send your data to any compatible third-party service.
-
+ 
 Integrate with your systems by configuring destinations and message templates.
+ 
+From the API perspective, this is the diagram flow: Workflows => Channel id(s) => Destination
 
-## Destinations [#destinations]
+## Why are there 3 different configurations? [#configurations]
+
+We have separated configurations to configure a customized notification so we can provide connection details/ to authenticate once(Destination), but create many customized templates for your outgoing events, and connect these with the destination.
+ 
+The Notifications API and configuration objects are product agnostic. Workflows is one of 3 products that use the Notifications API to send notifications events / integrate with third parties, along with [Error Inbox](https://docs.newrelic.com/docs/errors-inbox/error-external-services/) and others. Having a unified platform for notifications, you can, for example, set a single Slack or Jira destinations, and use them both in a Workflow and Error-Inbox configuration
+
+## Workflows [#workflows]
 
-Destinations are unique identifiers and connection details for third-party systems. Use destinations to integrate, send notifications, and share data between New Relic and your third-party systems.
+In a Workflow you can create triggers (filters) to decide when the workflow will run, can add “enrichments' (data that will be fetched from NRDB and appear in the outgoing events) and create channels.
 
-## Message templates [#message-templates]
+## Message templates (also known as channels) [#message-templates]
+
+Configure the eventual notification events and map New Relic data to your third-party services using message-templates. Channels (message templates) is a configuration object containing all the information required to send a notification event. “Destination” and “Properties” are the main building blocks of a Channel. A Channel enables users to customize their notification event data before it’s sent to your third-party destination.
```

Comment:
Either "A channel is..." or "Channels are..."

I'd like to establish the link between "message templates" and "channels" in the previous sentence, so that we're not parenthesizing again in this sentence.